modules/grpc/1.48.1.bcr.3/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/grpc/grpc/archive/refs/tags/v1.48.1.tar.gz",
+    "integrity": "sha256-MgNmZl0ZAnzah7I2jAOTkAajfgOIv9EJHI0qlvvJO9g=",
+    "strip_prefix": "grpc-1.48.1",
+    "patches": {
+        "adopt_bzlmod.patch": "sha256-+XSHXK0VjNIzdW+A3t2PYDKhtl/Ij/7hBQBkH42S2eE="
```

Comment:
to confirm, you merged the two patches of 1.48.1.bcr.2 together, right?

Reply:
Yes exactly. I merged the two patches and also included the one-line patch to make the `//third_party:protocol_compiler` target public.